### DISCUSSION TITLE
Is it intentional that the H1 on the list and page partials are drastically different sizes?

### DISCUSSION THREAD

**Discussion by simonhollingshead** [luizdepra/hugo-coder#776] [Q&A] [answered]
A page H1 is set to a size of 4.2rem (with line height 4.6rem).  Meanwhile, H1s elsewhere are 3.2rem (with a line height of 3.6rem).

You can see this quite clearly comparing the 'About' and 'Blog' pages on the example site.

Is this a stylistic decision or an error?  It makes it look like the list pages are missing an H1 and have an H2 at the top instead.

**Comment by luizdepra** [comment, marked as answer]
No, it is an error. :/
All page titles should have the same size.

Could you or someone open a PR to fix the issues? It is a good and simple first issue to contribute. :)
